db/schema.rb

```diff
@@ -340,6 +341,29 @@
     t.index ["publisher_id"], name: "index_potential_payments_on_publisher_id"
   end
 
+  create_table "potential_paypal_payments", id: :uuid, default: -> { "uuid_generate_v4()" }, force: :cascade do |t|
```

Comment:
I don't see a migration for this. Can you reset db/schema.rb and re-run your migrations?